ccmlib/node.py

```diff
@@ -1116,8 +1116,10 @@ def stress(self, stress_options=None, capture_output=False, **kwargs):
                 stress_options.extend(['-port', 'jmx=' + self.jmx_port])
         args = [stress] + stress_options
         try:
+            stdout_handle = kwargs.pop("stdout", subprocess.PIPE)
+            stderr_handle = kwargs.pop("stderr", subprocess.PIPE)
```

Comment:
but shouldn't they default to `subprocess.PIPE` if not given in kwargs?

Reply:
that's what the code does :)